readme.md

```diff
@@ -65,6 +66,16 @@ After this you can access all environment settings you have defined in one of yo
 console.log(process.env.APP_MY_ENV)
 ```
 
+## Automatic loading
+
+If you don't want to control the init process you can also import an automated version:
+
+```js
+import "universal-dotenv/auto"
```

Comment:
I would suggest using '/register' - that's what a few of these kind of modules do... e.g. in the Storybook world..